spec/codelog/command/pending_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe Codelog::Command::Pending do
+  describe '#run' do
+    let(:mock_file_outputter) { double(Codelog::Output::ReleaseFile) }
+
+    it 'calls' do
```

Comment:
```suggestion
    it 'adds the pending changes to the changelog' do
```
or something like that IMHO

Reply:
Works for me! I must have gotten distracted writing that description because I definitely didn't mean to submit that how it is lol